wnbd-client/client.cpp

```diff
@@ -316,6 +316,27 @@ DWORD execute_reset_opt(const po::variables_map& vm)
         safe_get_param<bool>(vm, "persistent"));
 }
 
+DWORD execute_install(const po::variables_map& vm)
+{
+    return CmdInstall(
+        safe_get_param<string>(vm, "filename").c_str());
+}
+
+void get_filename_args(
```

Comment:
Nit: let's rename this to "get_install_args". We might add other parameters, plus the filename description is too tied to this operation to be used in other scenarios.